scripts/spider.py

```
@@ -265,8 +276,17 @@ def get_page(url, session=None, use_cookie=False, use_proxy=False, module_name='
         # Add x-hostname header (required for Request Mirroring)
         headers['x-hostname'] = target_hostname
         
+        # Check if we need to force refresh CF bypass cookies
+        if cf_bypass_force_refresh:
```

Ruff (0.14.8)

Undefined name `cf_bypass_force_refresh`

(F821)